telemetry/definitions/commonDefinitions.json

```
@@ -1560,6 +1560,14 @@
             ],
             "description": "Type of save executed"
         },
+        {
+            "name": "scenarioType",
```

all types are types. don't need "type" in the name.

```suggestion
            "name": "scenario",
```